generator/src/lib.rs

```diff
@@ -1123,6 +1146,40 @@ pub fn generate_extension<'a>(
     };
     Some(q)
 }
+pub fn generate_define(define: &vkxml::Define) -> Tokens {
+    // TODO: Would rather leverage generate_constant
+    let name = constant_name(&define.name);
+    let ident = Ident::from(name.as_str());
+
+    if name == "NULL_HANDLE" {
+        quote!()
+    } else if let Some(value) = &define.value {
+        if let Ok(v) = str::parse::<u32>(value) {
+            quote!(pub const #ident: u32 = #v;)
+        } else {
+            quote!()
+        }
+    } else if let Some(c_expr) = &define.c_expression {
+        if define.defref.contains(&"VK_MAKE_VERSION".to_string())
+            && !define
```

Comment:
Can you move this into a variable and then do `&& !deprecated`?

Reply:
Moved it to the top-level:

```rust
    if name == "NULL_HANDLE" || deprecated {
        quote!()
```

Guess we don't want to generate any of these if they are deprecated ;)